src/gui/guiAnimatedImage.cpp

```diff
@@ -81,3 +63,11 @@ void GUIAnimatedImage::draw()
 		m_frame_time %= m_frame_duration;
 	}
 }
+
+
+void GUIAnimatedImage::setFrameIndex(s32 frame)
+{
+	s32 idx = std::max(frame, 0);
+	if (idx > 0 && idx < m_frame_count)
+		m_frame_idx = idx;
+}
```

Comment:
```suggestion
}

```

Reply:
We need an extra newline at the end of a file?  Why?

Reply:
Having an ending new line comes from unix standards, but computers are usually clever enough to work it out these days: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

![image](https://user-images.githubusercontent.com/2122943/76155034-5c8d6d00-60de-11ea-81ce-9710a07b21f6.png)